### PR TITLE
T1891

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/c30865ab631b28f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @k88hudson received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6761df35fdf0e66/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/c30865ab631b28f/output.txt

Total script time: 0.58 mins

Server is running on http://pr707.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/6761df35fdf0e66/output.txt

Total script time: 0.30 mins

File: src/timeline/media.js [hunk 1]
```diff
@@ -51,6 +51,11 @@ define( [ "core/trackevent", "core/track", "core/eventmanager",
 
     EventManagerWrapper( _this );
 
+    function onEditorMinimized( e ) {
+      _timebar.update( _zoom );
+      _tracksContainer.update();
```
Comment (mjschranz) [inline src/timeline/media.js:56]: It doesn't look like the actual tracks get updated.

File: src/timeline/track-container.js [hunk 2]
```diff
@@ -66,6 +66,8 @@ define( [ "core/logger", "util/dragndrop" ],
 
     function resetContainer() {
       _container.style.width = _media.duration * _zoom + "px";
+      _vScrollbar.update();
+      _hScrollbar.update();
```
Comment (mjschranz) [inline src/timeline/track-container.js:70]: Either what's mentioned above or these aren't updating them properly.

**Pull request merged**